spec/features/user_can_submit_posts_spec.rb

```diff
@@ -1,11 +1,22 @@
 require 'rails_helper'
 
-RSpec.feature "Timeline", type: :feature do
-  scenario "Can submit posts and view them" do
+RSpec.feature "Posts", type: :feature do
+  scenario "Can submit posts and view them without an image" do
     visit "/posts"
     click_link "New post"
     fill_in "Message", with: "Hello, world!"
     click_button "Submit"
     expect(page).to have_content("Hello, world!")
   end
+
+  scenario "can submit a post with an uploaded image included" do
+    visit "/posts"
+    click_link "New post"
+    fill_in "Message", with: "Image upload test!"
+    page.attach_file("Image", Rails.root + "spec/test_assets/test_image_1.png")
```

Comment:
Nice way of making the test act like it is uploading an image with a test image as well.